SystemServices/SystemServices.cpp

```diff
@@ -3722,10 +3724,19 @@ namespace WPEFramework {
                     (void*)&param, sizeof(param));
 
                 if (res == IARM_RESULT_SUCCESS) {
-                    if (param.curState == IARM_BUS_PWRMGR_POWERSTATE_ON)
+                    if (param.curState == IARM_BUS_PWRMGR_POWERSTATE_ON){
                         currentState = "ON";
-                    else if ((param.curState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY) || (param.curState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY_LIGHT_SLEEP) || (param.curState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY_DEEP_SLEEP))
-                        currentState = "STANDBY";
+                    }
+                    else if ((param.curState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY)) {
+			currentState = "STANDBY";
+		    }
+                    else if ((param.curState == IARM_BUS_PWRMGR_POWERSTATE_STANDBY_LIGHT_SLEEP)) {
```

Comment:
For STANDBY and LIGHT_SLEEP during set call u setting IARM_BUS_PWRMGR_POWERSTATE_STANDBY so during the get call it will return only STANDBY